src/lib/features/playground/playground-service.ts

```diff
@@ -11,12 +11,20 @@ import { FeatureStrategiesEvaluationResult } from 'lib/features/playground/featu
 import { ISegmentService } from 'lib/segments/segment-service-interface';
 import { FeatureConfigurationClient } from '../../types/stores/feature-strategies-store';
 import { generateObjectCombinations } from './generateObjectCombinations';
+import groupBy from 'lodash.groupby';
+import { omitKeys } from '../../util';
 
 type EvaluationInput = {
     features: FeatureConfigurationClient[];
     segments: ISegment[];
     featureProject: Record<string, string>;
     context: SdkContextSchema;
+    environment: string;
+};
+
+type AdvancedPlaygroundFeatureSchema = PlaygroundFeatureSchema & {
```

Comment:
This will be openapi schema once we define it, but for now I neded to add extra context and environment info to prepare UI specific read model